.github/workflows/unittest.yml

```diff
@@ -15,12 +15,16 @@ jobs:
       - uses: actions/checkout@v2
       - run: sudo apt-get update
       - run: sudo apt-get install -y cmake ccache build-essential
-      - run: sudo apt-get install -y maven
+      - run: sudo apt-get install -y maven clang-format
       - run: sudo apt-get install -y libboost-all-dev libcurl4-openssl-dev
       - run: sudo apt-get install -y libssl-dev
-      - run: sudo apt-get install -y libz-dev
+      - run: sudo apt-get install -y libz-dev python3-pip
+      - run: sudo pip install regex
       - name: Compile C++ unit tests
         run: |
           git submodule sync --recursive && git submodule update --init --recursive
           ./scripts/setup-ubuntu.sh 
           make release EXTRA_CMAKE_FLAGS="-DVELOX_ENABLE_ARROW=ON"
+      - name: Check formatting
+        run: |
+          ./scripts/check.py format commit
```

Comment:
I notice Velox uses below command to trigger format check and fix, is this aligned with that?

```
make format-check
make format-fix
```